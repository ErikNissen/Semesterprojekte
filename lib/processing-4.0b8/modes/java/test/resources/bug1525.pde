version https://git-lfs.github.com/spec/v1
oid sha256:5e4e224a4659f3830c33b801ce99dfe408d5df83288b159a79c610f6af8a0326
size 67
