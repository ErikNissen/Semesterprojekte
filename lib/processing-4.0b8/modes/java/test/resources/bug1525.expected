version https://git-lfs.github.com/spec/v1
oid sha256:4bfae59ba377c14033509f64487f559e5a3f4ecad7a0fffb91f2c872066bf332
size 761
